src/common/Platform.h

```diff
@@ -39,7 +39,7 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #elif defined(__linux__)
 #define PLATFORM_STRING "Linux"
 #elif defined(__native_client__)
-#define PLATFORM_STRING "Native Client"
+#define PLATFORM_STRING "NativeClient"
```

Comment:
I don't like it. But anyway it's not used anywhere

Reply:
We can also say “NaCl”. I want a single word because this can end up in version strings, and having one word per token makes parsing by third-party software easier.

Reply:
I've set `NaCl`.